common/filestore_utils.py

```diff
@@ -71,7 +71,7 @@ def rsync(  # pylint: disable=too-many-arguments
         options=None,
         parallel=False):
     """Syncs |source| and |destination| folders. |gsutil_options| and |parallel|
-    are only used in gsutil."""
+    are only used when by the gsutil implementation."""
```

Comment:
`|gsutil_options| and |parallel| are only used by the gsutil implementation.`

Reply:
Nvm. :)